build.cmd

```diff
@@ -80,6 +82,11 @@ for %%T IN (Restore %MSBuildBuildTarget%, BuildModernVsixPackages) do (
   )
 )
 
+REM Run copy as a final step after all the product components are built
+if /I "%CopyOutputArtifacts%" == "true" (
+  call build\Scripts\CopyOutput.cmd
```

Comment:
Wouldn't `call build\Scripts\CopyOutput.cmd` get called from calling script i.e. build.cmd, making this path relative to build.cmd?